### PR TITLE
walkthrough-2 (Portfolio)

### PR DESCRIPTION
**Pull request by dmosc**

### CODE REVIEW

File: portfolio/src/main/webapp/static/constants/projects.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        const description = document.createElement("p");
+
+        title.innerHTML = project.title;
+        title.classList.add("badge", "badge-secondary");
+        description.innerHTML = project.description;
+        
+        const project_to_append = document.createElement("li");
+
+        project_to_append.classList.add("list-group-item", "text-wrap");
+        project_to_append.style = "width: 100%;";
```
Comment (anniefu) [inline portfolio/src/main/webapp/static/constants/projects.js]: It'd probably be best to consolidate all your styling to your .css files. That way in the future if someone was trying to adjust something, it's more transparent where the styling is coming from.

I'd recommend adding a class/id to your project_to_append element that your CSS file can use.

File: portfolio/src/main/webapp/static/constants/social_networks.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    for (const [key, value] of Object.entries(networks)) {
+        const network_to_append = document.createElement("a");
+        
+        network_to_append.href = value;
+        network_to_append.rel = "noopener noreferrer";
+        network_to_append.target = "_blank";
+
+        const icon = document.createElement("i");
+        icon.classList.add('fab', `fa-${key}`);
+        icon.style = "width: 20px; height: 20px;";
```
Comment (anniefu) [inline portfolio/src/main/webapp/static/constants/social_networks.js]: Same recommendation as above, to put the styling in the CSS file.

**Review by anniefu — APPROVED** [review]
Hey Oscar, good start!

I left a suggestion that I think would be good to take action on, but aren't critical. I'm gonna give approval right now so that if you push a new commit to fix them, you can merge immediately after.

File: portfolio/src/main/webapp/static/constants/projects.js (hunk 1 above)
Comment (dmosc) [reply portfolio/src/main/webapp/static/constants/projects.js]: That's a really good suggestion. That approach centralizes better the CSS styling.
I'll go ahead and resolve it in a new commit.

**Review by ssmall — APPROVED** [review]
Looks good! I second Annie's comments about styling.

**Pull request merged**